src/sentry/db/models/fields/bounded.py

```diff
@@ -110,3 +110,6 @@ class BoundedBigAutoField(BoundedAutoField):
 add_introspection_rules([], ["^sentry\.db\.models\.fields\.bounded\.BoundedBigIntegerField"])
 add_introspection_rules([], ["^sentry\.db\.models\.fields\.bounded\.BoundedPositiveIntegerField"])
 add_introspection_rules([], ["^sentry\.db\.models\.fields\.pickle\.UnicodePickledObjectField"])
```

Comment:
so i think this i no longer valid as i couldnt find it in local code, but i did fix the missing ones